makefile

```diff
@@ -11,6 +11,12 @@ uname := $(shell uname -s)
 BUILDX_PLATFORM ?= linux/amd64,linux/arm64,linux/arm,linux/ppc64le,linux/s390x
 DOCKER_ORGS ?= aquasec public.ecr.aws/aquasecurity
 GOARCH ?= $@
+KUBECTL_VERSION ?= 1.28.7
+KUBECTL_SUM_arm64 ?= $(shell curl -L "https://dl.k8s.io/release/v$(KUBECTL_VERSION)/bin/linux/arm64/kubectl.sha256")
+KUBECTL_SUM_amd64 ?= $(shell curl -L "https://dl.k8s.io/release/v$(KUBECTL_VERSION)/bin/linux/amd64/kubectl.sha256")
+KUBECTL_SUM_s390x ?= $(shell curl -L "https://dl.k8s.io/release/v$(KUBECTL_VERSION)/bin/linux/s390x/kubectl.sha256")
+KUBECTL_SUM_ppc64le ?= $(shell curl -L "https://dl.k8s.io/release/v$(KUBECTL_VERSION)/bin/linux/ppc64le/kubectl.sha256")
+ARCH ?= $(shell docker info --format '{{.ClientInfo.Arch}}')
```

Comment:
Not all version of docker support this field:

```
$ docker info --format '{{.ClientInfo.Arch}}'

template: :1:13: executing "" at <.ClientInfo.Arch>: can't evaluate field Arch in type *system.clientInfo
```

How about get arch via `go env GOARCH` ?